meegkit/trca.py

```diff
@@ -71,6 +74,74 @@ def trca(X):
     return W_best
 
 
+def trca_regul(X, regul):
+    """Task-related component analysis.
+
+    This function implements a variation of the method described in [1].
```

Comment:
```suggestion
    This function implements a variation of the method described in [1]_.
```

Can you also reference the Barachant document here?